File: ui/Planilla/src/bean/Empleado.java
package bean;

public class Empleado {
int id,idDomiciliado,idEstadoCivil,idNivelEducativo,idDepartamento,idProvincia,idDistrito;
String Dni,RUC,Nombres,ApellidoPaterno,ApellidoMaterno,FechaNacimiento,Direccion,Correo,Celular,Passwordd;

public int getId() {
	return id;
}
public void setId(int id) {
	this.id = id;
}

public int getIdDomiciliado() {
	return idDomiciliado;
}
public void setIdDomiciliado(int idDomiciliado) {
	this.idDomiciliado = idDomiciliado;
}
public int getIdEstadoCivil() {
	return idEstadoCivil;
}
public void setIdEstadoCivil(int idEstadoCivil) {
	this.idEstadoCivil = idEstadoCivil;
}
public int getIdNivelEducativo() {
	return idNivelEducativo;
}
public void setIdNivelEducativo(int idNivelEducativo) {
	this.idNivelEducativo = idNivelEducativo;
}
public int getIdDepartamento() {
	return idDepartamento;
}
public void setIdDepartamento(int idDepartamento) {
	this.idDepartamento = idDepartamento;
}
public int getIdProvincia() {
	return idProvincia;
}
public void setIdProvincia(int idProvincia) {
	this.idProvincia = idProvincia;
}
public int getIdDistrito() {
	return idDistrito;
}
public void setIdDistrito(int idDistrito) {
	this.idDistrito = idDistrito;
}
public String getDni() {
	return Dni;
}
public void setDni(String dni) {
	Dni = dni;
}
public String getRUC() {
	return RUC;
}
public void setRUC(String rUC) {
	RUC = rUC;
}
public String getNombres() {
	return Nombres;
}
public void setNombres(String nombres) {
	Nombres = nombres;
}
public String getApellidoPaterno() {
	return ApellidoPaterno;
}
public void setApellidoPaterno(String apellidoPaterno) {
	ApellidoPaterno = apellidoPaterno;
}
public String getApellidoMaterno() {
	return ApellidoMaterno;
}
public void setApellidoMaterno(String apellidoMaterno) {
	ApellidoMaterno = apellidoMaterno;
}
public String getFechaNacimiento() {
	return FechaNacimiento;
}
public void setFechaNacimiento(String fechaNacimiento) {
	FechaNacimiento = fechaNacimiento;
}
public String getDireccion() {
	return Direccion;
}
public void setDireccion(String direccion) {
	Direccion = direccion;
}
public String getCorreo() {
	return Correo;
}
public void setCorreo(String correo) {
	Correo = correo;
}
public String getCelular() {
	return Celular;
}
public void setCelular(String celular) {
	Celular = celular;
}
public String getPasswordd() {
	return Passwordd;
}
public void setPassword(String passwordd) {
	Passwordd = passwordd;
}

public void setEmpleado(String[] aRegistro){
	if (aRegistro == null) return;
	id = Integer.parseInt(aRegistro[0]);
	Dni = aRegistro[1];
	RUC = aRegistro[2];
	Nombres= aRegistro[3];
	ApellidoPaterno = aRegistro[4];
	ApellidoMaterno= aRegistro[5];
	FechaNacimiento= aRegistro[6];
	Direccion= aRegistro[7];
	Correo= aRegistro[8];
	Celular= aRegistro[9];
	Passwordd= aRegistro[10];
	idDomiciliado= Integer.parseInt(aRegistro[11]);
	idEstadoCivil= Integer.parseInt(aRegistro[12]);
	idNivelEducativo= Integer.parseInt(aRegistro[13]);
	idDepartamento= Integer.parseInt(aRegistro[14]);
	idProvincia= Integer.parseInt(aRegistro[15]);
	idDistrito= Integer.parseInt(aRegistro[16]);
	
}
	
}

File: bean/Empleado.cs
using System.Data;

namespace appPlanilla_IA_AMV.bean
{
    internal class Empleado
    {
        #region Propiedades
        public int id { get; set; }
        public string Dni { get; set; }
        public string Ruc { get; set; }
        public string Nombres { get; set; }
        public string ApeliidoPaterno { get; set; }
        public string ApellidoMaterno { get; set; }
        public string FechaNacimiento { get; set; }
        public string Direccion { get; set; }
        public string Correo { get; set; }
        public string Celular { get; set; }
        public string Passwordd { get; set; }
        public int idDomicilio { get; set; }
        public int idEstadoCivil { get; set; }
        public int idNivelEducativo { get; set; }
        public int idDepartamento { get; set; }
        public int idProvincia { get; set; }
        public int idDistrito { get; set; }
        #endregion

        #region Mètodos

        public void setRegistro(DataRow dr)
        {
            if (dr == null) return;

            id = int.Parse(dr["id"].ToString());
            Dni = dr["Dni"].ToString();
            Ruc = dr["Ruc"].ToString();
            Nombres = dr["Nombres"].ToString();
            ApeliidoPaterno = dr["ApellidoPaterno"].ToString();
            ApellidoMaterno = dr["ApellidoMaterno"].ToString();
            FechaNacimiento = dr["FechaNacimiento"].ToString();
            Direccion = dr["FechaNacimiento"].ToString();
            Correo = dr["Correo"].ToString();
            Celular = dr["Celular"].ToString();
            Passwordd = dr["Passwordd"].ToString();
            idDomicilio = int.Parse(dr["idDomicilio"].ToString());
            idEstadoCivil = int.Parse(dr["idEstadoCivil"].ToString());
            idNivelEducativo = int.Parse(dr["idNivelEducativo"].ToString());
            idDepartamento = int.Parse(dr["idDepartamento"].ToString());
            idProvincia = int.Parse(dr["idProvincia"].ToString());
            idDistrito = int.Parse(dr["idDistrito"].ToString());
        }
            #endregion      

        // cambios a subir
    }
}
